docs/conf.py

```diff
@@ -234,6 +234,14 @@ def linkcode_resolve(domain: str, info: Dict[str, Any]) -> Optional[str]:
 hoverxref_tooltip_theme = ["tooltipster-custom"]
 hoverxref_tooltip_lazy = True
 
+# these have to match the keys on intersphinx_mapping, and those projects must be hosted on read the docs.
+hoverxref_intersphinx = [
```

Comment:
Could you mention this config value somewhere next to `intersphinx_mapping` as well, so that this won't be missed when adding new intersphinx mappings in the future?

Reply:
https://github.com/DisnakeDev/disnake/pull/788/commits/199811844952e66d90325de0f7cf0b2439fc6194